src/main/filter.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * To add a new filter, create a filter function that matches the type FilterFunction and then add the mapping to the FILTER_MAP
+ */
+
+import { MEMBER_MAP } from "./members";
+
+type FilterMap = { [onestopFilterName: string]: FilterFunction };
+
+const COMMA_DELIMITER: string = ",";
+const BROS_GENDER: string = "Male";
+const SIS_GENDER: string = "Female";
+// Maps a filter's name on the Onestop to its corresponding filter function
+const FILTER_MAP: FilterMap = {
```

Comment:
can we also add "minus moms" and "minus dads"? those are used for mbs every week

Reply:
Added the `Minus Moms` and `Minus Dads` filters. Was pretty straightforward so hopefully adding additional filters shouldn't be too complicated